### PR TITLE
Revert csv sniffing

### PR DESCRIPTION
**Pull request by atvaccaro**
# Description
_Describe your changes and why you're making them. Please include the context, motivation, and relevant dependencies._

It turns out that the CSV sniffing regex blows up on specific files, but we can't really just sniff a sample in case of doublequote issues. I'm going to revert this, and just specifically handle commas and tabs within the `'excel'` dialect (the default).

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

## How has this been tested?
_Include commands/logs/screenshots as relevant._

## Post-merge follow-ups
_Document any actions that must be taken post-merge to deploy or otherwise implement the changes in this PR (for example, running a full refresh of some incremental model in dbt). If these actions will take more than a few hours after the merge or if they will be completed by someone other than the PR author, please create a dedicated follow-up issue and link it here to track resolution._

- [ ] No action required
- [x] Actions required (specified below)

Need to clear the stalled stop_times and re-run external tables.

### CODE REVIEW

File: airflow/plugins/operators/gtfs_csv_to_jsonl_hourly.py [hunk 1]
```diff
@@ -90,8 +90,7 @@ def parse_individual_file(
     lines = []
     try:
         with fs.open(input_file.path, newline="", mode="r", encoding="utf-8-sig") as f:
-            # read the whole file so we see any doublequote usage, mainly
-            dialect = csv.Sniffer().sniff(f.read())
+            dialect = csv.Sniffer().sniff(f.readline())
```
Comment (lauriemerrell) [inline airflow/plugins/operators/gtfs_csv_to_jsonl_hourly.py:93]: Does this also need to be deleted?

Comment (atvaccaro) [reply airflow/plugins/operators/gtfs_csv_to_jsonl_hourly.py:93]: yes I just realized it was a separate PR and was about to push

**Review by lauriemerrell — APPROVED** [review]
Do we want a follow up ticket to handle tabs through some other manner? You reference Excel dialect, I thought that by reverting this all as-is we will continue to fail on the tab-delimited feed

**Comment by lauriemerrell** [comment]
I will reopen #2484?

**Pull request merged**